### PR TITLE
API Stop using deprecated API

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10545

### CODE REVIEW

File: tests/CommentsGridFieldActionTest.php [hunk 1]
```diff
@@ -40,6 +41,9 @@ class CommentsGridFieldActionTest extends SapphireTest
     protected function setUp(): void
     {
         parent::setUp();
+        if (Deprecation::isEnabled()) {
+            $this->markTestSkipped('Test calls deprecated code');
+        }
         $this->list = new DataList(Team::class);
```
Comment (sabina-talipova) [inline tests/CommentsGridFieldActionTest.php:47]: Could we also mark as deprecated `$list` property https://github.com/silverstripe/silverstripe-comments/blob/b771bb624afc8c8deaa368883a6044e331f1a2e0/tests/CommentsGridFieldActionTest.php#L33

File: tests/CommentsGridFieldActionTest.php [hunk 2]
```diff
@@ -40,6 +41,9 @@ class CommentsGridFieldActionTest extends SapphireTest
...
         $config = CommentsGridFieldConfig::create()->addComponent(new GridFieldDeleteAction());
         $this->gridField = new CommentsGridField('testfield', 'testfield', $this->list, $config);
```
Comment (sabina-talipova) [inline tests/CommentsGridFieldActionTest.php:49]: Should we leave `setUp` method to `$gridField ` initialisation?
And mark as deprecated `$form` https://github.com/silverstripe/silverstripe-comments/blob/b771bb624afc8c8deaa368883a6044e331f1a2e0/tests/CommentsGridFieldActionTest.php#L50

**Review by sabina-talipova — CHANGES_REQUESTED** [review]

File: tests/CommentsGridFieldActionTest.php (hunk 1 above)
Comment (emteknetnz) [reply tests/CommentsGridFieldActionTest.php:47]: It's a unit test, no need to deprecate things

File: tests/CommentsGridFieldActionTest.php (hunk 2 above)
Comment (emteknetnz) [reply tests/CommentsGridFieldActionTest.php:49]: It's a unit test, no need to deprecate things

**Pull request merged**